examples/step-37/doc/results.dox

```diff
@@ -11,122 +11,193 @@ solution through both isocontours and volume rendering:
-Time solve (6 iterations)  (CPU/wall) 0.02s/0.01133s
+Total setup time               (wall) 0.00678205s
+Time solve (6 iterations)  (CPU/wall) 0.005616s/0.00561595s
 
 Cycle 4
 Number of degrees of freedom: 16641
-System matrix memory consumption:     0.7343 MB.
-Multigrid objects memory consumption: 1.925 MB.
-Total setup time               (wall) 0.02938s
-Time solve (6 iterations)  (CPU/wall) 0.068s/0.03312s
+Total setup time               (wall) 0.0241671s
+Time solve (6 iterations)  (CPU/wall) 0.019543s/0.0195441s
 
 Cycle 5
 Number of degrees of freedom: 66049
-System matrix memory consumption:     2.856 MB.
-Multigrid objects memory consumption: 7.435 MB.
-Total setup time               (wall) 0.1128s
-Time solve (6 iterations)  (CPU/wall) 0.228s/0.09577s
+Total setup time               (wall) 0.0967851s
+Time solve (6 iterations)  (CPU/wall) 0.07457s/0.0745709s
 
 Cycle 6
 Number of degrees of freedom: 263169
-System matrix memory consumption:     11.28 MB.
-Multigrid objects memory consumption: 29.3 MB.
-Total setup time               (wall) 0.4553s
-Time solve (6 iterations)  (CPU/wall) 1.272s/0.3955s
+Total setup time               (wall) 0.346374s
+Time solve (6 iterations)  (CPU/wall) 0.260042s/0.265033s
 @endcode
 
 As in step-16, we see that the number of CG iterations remains constant with
-increasing number of degrees of freedom. We can also see that the various
-objects we have to store for the multigrid method on the individual levels of
-our mesh together make up more than twice as much as the matrix on the finest
-level. For the present example, about half the memory consumption of the
-multigrid objects are the level transfer matrices, and the other half is
-consumed by the matrix-free objects (and there, mainly the indices and the
-variable coefficient).
+increasing number of degrees of freedom. A constant number of iterations
+(together with optimal computational properties) means that the computing time
+approximately quadruples as the problem size quadruples from one cycle to the
+next. A look at the memory consumption reveals that the code is also very
+efficient in terms of storage. Around 2-4 million degrees of freedom fit into
+1 GB of memory. An interesting fact is that solving one linear system is
+cheaper than the setup, despite not building a matrix (approximately half of
+which is spent in the DoFHandler::distribute_dofs() and
+DoFHandler::distributed_mg_dofs() calls). This shows the high efficiency of
+this approach, but also that the deal.II data structures are quite expensive
+to set up and the setup cost must be amortized over several system solves.
 
 Not much changes if we run the program in three spatial dimensions, with the
-exception that the multilevel objects now take up some more memory (because
-the level transfer matrices are denser) and the computing times are somewhat
-larger:
+exception that problem sizes grow by a factor eight when refining the mesh and
```

Comment:
oh, i see. Maybe just to remind the reader mention that it's global refinement? (although i was reading the changes only, thus I did not pick this up).